int64 cmd_type
float64[] cmd_data
---
float64[] obs
